data_analysis/profit_analysis.py

```diff
@@ -3,18 +3,14 @@
 for each prediction.
 """
 
-from data_analysis.price_prediction import model_arima
-from get_data.get_data import get_current_value
 
-
-def sell_today(n_days, value, predictions):
+def sell_today(value, predictions):
     """Returns a boolean value for to indicate whether to sell today."""
     return value < max(predictions.predicted_value)
```

Comment:
This should be > as we want the value to be higher than our predictions

Reply:
Thank you for spotting this!